index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+  <script src="index.js"></script>
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <div class="header-1 desktop">
+      <ul class="menu-hide">
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="header-2 desktop">
+      <div class="logo"><a href="index.html"><img src="img/logo.png" alt="logo"></a></div>
+      <nav>
+        <ul class="menu" id="menu">
+          <li class="mobile"><img src="img/Icon-close.png" alt="close" id="close"></li>
+          <li><a href="about.html" class="active">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">IT Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+
+  </header>
+  <section class="headline">
+    <div class="menu-mobile mobile">
+      <img src="img/menu.png" alt="menu" id="hamburger">
+    </div>
+    <h3>Learn Code by doing !</h3>
+    <h1>MEETING SOFTWARE WEB AND VIDEO CODE</h1>
+    <p class="text-1">Go ahead, give it a try. Our hands-on learning environment means you'll be writing real code from
+      your very first lesson.
+      This short quiz will sort you out.
+
+    </p>
+    <h2 class="date">2023.10.15(THU) ~ 16(FRI)</h2>
+    <p class="text-2">@Technology-Full Stack, Development</p>
+    
+  </section>
+  <section class="program desktop anim">
+
+    <h2>Main Program</h2>
+    <div class="underline"></div>
+    <div class="block anim">
+      <div class="anim">
+        <img src="img/lecture.png" alt="lecture">
+        <h3>Lecture</h3>
+        <p>Learn to code . Build projects. Graduates have gotten jobs at tech companies</p>
+      </div>
+      <div>
+        <img src="img/projection-screen.png" alt="projection">
+        <h3>Projection</h3>
+        <p>Learn to code . Build projects. Graduates have gotten jobs at tech companies</p>
+      </div>
+      <div>
+        <img src="img/discussion.png" alt="discussion">
+        <h3>Discussion</h3>
+        <p>Learn to code . Build projects. Graduates have gotten jobs at tech companies</p>
+      </div>
+      <div class="active anim">
+        <img src="img/workshop1.png" alt="workshop">
+        <h3>Workshop</h3>
+        <p>Learn to code . Build projects. Graduates have gotten jobs at tech companies</p>
+      </div>
```

Comment:
- [ ] [OPTIONAL] I think the white border around the card was meant to show up when hovering over it. It would be great if you implement it in such a way that it appears when hovering over the cards. 🤓️

<img src="https://user-images.githubusercontent.com/17243325/233121708-4a7ef0aa-963c-4308-819c-78d84729e296.png" width="300">